resources/qml/Cura.qml

```diff
@@ -885,6 +875,49 @@ UM.MainWindow
         }
     }
 
+    Cura.RenameDialog
+    {
+        id: createNewQualityDialog
+        title: catalog.i18nc("@title:window", "Save Custom Profile")
+        objectPlaceholder: catalog.i18nc("@textfield:placeholder", "New Custom Profile")
+        explanation: catalog.i18nc("@info", "Custom profile name:")
+        extraInfo:
+        [
+            UM.ColorImage
+            {
+                width: UM.Theme.getSize("message_type_icon").width
+                height: UM.Theme.getSize("message_type_icon").height
+                source: UM.Theme.getIcon("Information")
+                color: UM.Theme.getColor("text")
+            },
+            Column
+            {
+                UM.Label
+                {
+                    text: catalog.i18nc
+                    (
+                        "@label %i will be replaced with a profile name",
+                        "<b>Only user changed settings will be saved in the custom profile.</b><br/>" +
+                        "For materials that support it, the new custom profile will inherit properties from <b>%1</b>."
+                    ).arg(Cura.MachineManager.activeQualityOrQualityChangesName)
+                    wrapMode: Text.WordWrap
+                    width: parent.parent.width - 2 * UM.Theme.getSize("message_type_icon").width
+                }
+                Cura.TertiaryButton
+                {
+                    text: "Learn more about Cura print profiles"
+                    iconSource: UM.Theme.getIcon("LinkExternal")
+                    isIconOnRightSide: true
+                    leftPadding: 0
+                    rightPadding: 0
+                    onClicked: Qt.openUrlExternally("https://support.ultimaker.com/s/article/1667337576882")
```

Comment:
Isn't there a campaign link we should use for this?

Reply:
@MariMakes

Reply:
The link probably needs to be https://support.makerbot.com/s/article/1667337576882
We'll have to ask if it can be updated.

Reply:
Fixed in cloudflare with a redirect